Automattic/wp-calypso
# Theme Discovery: Add placeholder to collection carousels

Fixes https://github.com/Automattic/wp-calypso/issues/82847

## Proposed Changes

* Add pulsating placeholders to `ThemeCollection` while we are fetching themes.

<img width="1233" alt="Screenshot 2023-10-27 at 15 15 48" src="https://github.com/Automattic/wp-calypso/assets/2070010/fe82e5f5-58a9-4068-bcd4-ab1bcaf33ca4">


## Testing Instructions



* Open `/themes?flags=themes/discovery-lots` in incognito.
* Check that themes load as placeholders.
* Ensure the UI is not too twitchy (but a little bit is to be expected).

## Pre-merge Checklist
